.github/CONTRIBUTING.md

```diff
@@ -10,6 +10,7 @@ We accept pull requests for bug fixes and features where we've discussed the app
 
 Please do:
 
+* check existing issues to verify that the bug or feature request has not already been submitted
```

Comment:
Do we want to change `existing issues` to actually link to the issues page?

Reply:
Sure, we could link `bug` and `feature request` to go to their respective labels! Most people will want to check one or the other